src/test/java/co/rsk/federate/signing/hsm/advanceblockchain/HsmBookkeepingClientImplTest.java

```diff
@@ -486,21 +490,22 @@ private ObjectNode buildVersionRequest() {
     }
 
     private List<Block> buildBlocks() {
-        BlockHeader block1Header = blockHeaderBuilder.setNumber(1).build();
-        BlockHeader block2Header = blockHeaderBuilder.setNumber(2).build();
-        BlockHeader block3Header = blockHeaderBuilder.setNumber(3).build();
-
-        List<BlockHeader> block1Brothers = Arrays.asList(
-            blockHeaderBuilder.setNumber(101).build(),
-            blockHeaderBuilder.setNumber(102).build()
+        // 3 Blocks, 1 Brother Each For Block 2 And Block 1
+        List<BlockHeader> block1Uncles = Arrays.asList(
+            blockHeaderBuilder.setNumber(101).setParentHashFromKeccak256(TestUtils.createHash(0)).build(),
```

Comment:
```suggestion
            blockHeaderBuilder.setNumber(101).setParentHashFromKeccak256(block1.getHash()).build(),
```
That would make it a lot clearer

Reply:
Updated